moveit_ros/planning/planning_scene_monitor/src/current_state_monitor.cpp

```diff
@@ -251,13 +251,13 @@ bool CurrentStateMonitor::waitForCurrentState(const rclcpp::Time& t, double wait
 
 bool CurrentStateMonitor::waitForCompleteState(double wait_time) const
 {
-  double slept_time = 0.0;
-  double sleep_step_s = std::min(0.05, wait_time / 10.0);
-  rclcpp::Duration sleep_step(sleep_step_s);
-  while (!haveCompleteState() && slept_time < wait_time)
+  const double sleep_step_s = std::min(0.05, wait_time / 10.0);
+  const auto sleep_step_duration = rclcpp::Duration::from_seconds(sleep_step_s);
+  double slept_time_s = 0.0;
+  while (!haveCompleteState() && slept_time_s < wait_time)
   {
-    rclcpp::sleep_for(sleep_step.to_chrono<std::chrono::nanoseconds>());
-    slept_time += sleep_step_s;
+    rclcpp::sleep_for(sleep_step_duration.to_chrono<std::chrono::nanoseconds>());
+    slept_time_s += sleep_step_s;
```

Comment:
Nit about variable naming. I know ROS is "by default" SI units and the base unit for time is second and that should always be the implicit default. Yet, I find it much easier to read code and API if the units are documented as part of the variable names. In this case, it would mean that `wait_time` would be `wait_time_s` and `sleep_step_duration` would be `sleep_step_duration_s`. Thoughts?

Reply:
I didn't thought this before but really liked it 10/10 will use when I need

Reply:
sleep_step_duration is a Duration object and is not specifically in the unit seconds, it is in whatever unit you want.  I will change wait_time though.

Reply:
+1, Google style is to include units